### DISCUSSION TITLE
Remix SPA Mode support

### DISCUSSION THREAD

**Discussion by nabe-yu** [mantinedev/mantine#6783] [Q&A]
I am trying to use Remix SPA Mode.
When I installed Mantine and tried using it, the styles were applied when running 'npm run dev', but not when running 'npm run build'.
Can I use Remix SPA Mode with Mantine?   
I'm new to React, thank you in advance.

**Comment by rtivital** [comment]
There isn't much difference between SPA and not SPA modes in remix from Mantine perspective. It should work fine with both modes unless Remix has issues with styles loading or you configured something incorrectly.

**Reply by nabe-yu** [reply]
Thanks for the quick response. I will look into the build configurations.
